terraform/common/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 0.12.29"
```

Comment:
Do you have to switch terraform versions to apply common or app? Might as well update the version here.

Reply:
`>= 0.12.20` was satisfied when running with `0.13.1`, but I'm happy to make the minimum version consistent across `common` and `app`